### PR TITLE
Hovenweep doc

### LINKED ISSUE #526: Accessing -onprem datasets on Tallgrass
I am attempting to access 'nhm-v1.1-c404-bc-byHW-onprem' using hytest workflows while in an interactive session on Tallgrass. While I am not on the HYTEST server, I have a (nearly) equivalent environment (just a few extra packages) that I'm using for this work. A code snippet that's basically copied from the HYTEST example notebooks:

`hytest_cat = intake.open_catalog("https://raw.githubusercontent.com/hytest-org/hytest/main/dataset_catalog/hytest_intake_catalog.yml")

cat = hytest_cat['nhm-v1.1-c404-bc-catalog']
dataset = 'nhm-v1.1-c404-bc-byHW-onprem'

ds = cat[dataset].to_dask()` 

when I run this I get "FileNotFoundError: [Errno 2] No such file or directory: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.1/c404-bc_byHW/nhm_v1.1_c404-bc_byHW_combined.json/.zmetadata'"

I have also experienced this error when accessing CONUS404 data, but used the dataset with the -osn suffix as a workaround. What's going on here? My understanding from the docs is if you're on a USGS computer, it's suggested to use the -onprem data rather than the other sources. And in this particular case, the nhm data is only available 'onprem'.

### PR DESCRIPTION
**Pull request by amsnyder**
# Pull Request Code Review

## Overview
- Update intake catalog to use different naming convention for datasets stored on Hovenweep (`-onprem-hw`) so they are not confused with datasets stored on Tallgrass/Denali (`on-prem`).
- Update HyTEST intake catalog readme to describe how to differentiate files stored on Hovenweep from those on Tallgrass/Denali

What is the purpose of this request? Why? 1-2 sentences
Please let me know if they updated text makes this points above clear.

## Links

- *Issues*: Closes #526

### CODE REVIEW

**Comment by amsnyder** [comment]
@putnamannie if you are familiar with how to review Gitub pull requests, feel free to take a look and see if you think these changes make the distinction clear.

**Review by putmanannie — COMMENTED** [review]
The change to the naming convention will clear up a lot of trouble. Thanks! I also suggest updating the last sentence of each of the description to something like 'This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible through the USGS on-premise computer Hovenweep', as the current description could still cause confusion.

File: dataset_catalog/subcatalogs/conus404-catalog.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 sources:
 
-  conus404-hourly-onprem:
+  conus404-hourly-onprem-hw:
     driver: zarr
     description: "CONUS404 Hydro Variable subset, 43 years of hourly data on Caldera. These files were created wrfout model output files (see ScienceBase data release for more details: https://doi.org/10.5066/P9PHPK4F). This dataset is stored on USGS on-premise Caldera storage and is only accessible through USGS on-premise computing resources."
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/conus404-catalog.yml:5]:
```suggestion
    description: "CONUS404 Hydro Variable subset, 43 years of hourly data on Caldera. These files were created wrfout model output files (see ScienceBase data release for more details: https://doi.org/10.5066/P9PHPK4F). This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/conus404-catalog.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ sources:
         client_kwargs:
           endpoint_url: https://usgs.osn.mghpcc.org/
 
-  conus404-daily-diagnostic-onprem:
+  conus404-daily-diagnostic-onprem-hw:
     driver: zarr
     description: "CONUS404 43 years of daily diagnostic output (maximum, minimum, mean, and standard deviation) for water vapor (Q2), grid-scale precipitation (RAINNC), skin temperature (SKINTEMP), wind speed at 10 meter height (SPDUV10), temperature at 2 meter height (T2), and U- and V-component of wind at 10 meters with respect to model grid (U10, V10). These files were created wrfxtrm model output files (see ScienceBase data release for more details: https://doi.org/10.5066/P9PHPK4F). This dataset is stored on USGS on-premise Caldera storage and is only accessible through USGS on-premise computing resources."
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/conus404-catalog.yml:33]:
```suggestion
    description: "CONUS404 43 years of daily diagnostic output (maximum, minimum, mean, and standard deviation) for water vapor (Q2), grid-scale precipitation (RAINNC), skin temperature (SKINTEMP), wind speed at 10 meter height (SPDUV10), temperature at 2 meter height (T2), and U- and V-component of wind at 10 meters with respect to model grid (U10, V10). These files were created wrfxtrm model output files (see ScienceBase data release for more details: https://doi.org/10.5066/P9PHPK4F). This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/conus404-catalog.yml [hunk 3]
```diff
@@ -56,7 +56,7 @@ sources:
         client_kwargs:
           endpoint_url: https://usgs.osn.mghpcc.org/
 
-  conus404-daily-onprem:
+  conus404-daily-onprem-hw:
     driver: zarr
     description: "CONUS404 43 years of daily values for subset of model output variables derived from hourly values. This dataset is stored on USGS on-premise Caldera storage and is only accessible through USGS on-premise computing resources."
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/conus404-catalog.yml:61]:
```suggestion
    description: "CONUS404 43 years of daily values for subset of model output variables derived from hourly values. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/conus404-catalog.yml [hunk 4]
```diff
@@ -84,7 +84,7 @@ sources:
         client_kwargs:
           endpoint_url: https://usgs.osn.mghpcc.org/
           
-  conus404-monthly-onprem:
+  conus404-monthly-onprem-hw:
     driver: zarr
     description: "CONUS404 43 years of monthly values for subset of model output variables derived from daily values. This dataset is stored on USGS on-premise Caldera storage and is only accessible through USGS on-premise computing resources."
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/conus404-catalog.yml:89]:
```suggestion
    description: "CONUS404 43 years of monthly values for subset of model output variables derived from daily values. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/conus404-catalog.yml [hunk 5]
```diff
@@ -112,7 +112,7 @@ sources:
         client_kwargs:
           endpoint_url: https://usgs.osn.mghpcc.org/
 
-  conus404-hourly-ba-onprem:
+  conus404-hourly-ba-onprem-hw:
     driver: zarr
     description: "CONUS404 bias-adjusted temperature and precipitation data (see ScienceBase data release for more details: https://doi.org/10.5066/P9JE61P7). 43 years of hourly data on Hovenweep’s Caldera storage. This dataset is stored on USGS on-premise disk storage for the Hovenweep supercomputer, and is only accessible in that computing environment."
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/conus404-catalog.yml:117]:
```suggestion
    description: "CONUS404 bias-adjusted temperature and precipitation data (see ScienceBase data release for more details: https://doi.org/10.5066/P9JE61P7). 43 years of hourly data on Hovenweep’s Caldera storage. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml [hunk 6]
```diff
@@ -1,6 +1,6 @@
 sources:
 
-  nhm-v1.0-daymet-byHRU-onprem:
+  nhm-v1.0-daymet-byHRU-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.0 model output variables using byHRU calibrated parameters with Daymet version 3 forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml:5]:
```suggestion
    description: "National Hydrologic Model version 1.0 model output variables using byHRU calibrated parameters with Daymet version 3 forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml [hunk 7]
```diff
@@ -9,7 +9,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.0/dm_byHRU/nhm_v1.0_dm_byHRU_combined.json'
 
-  nhm-v1.0-daymet-byHW-musk-onprem:
+  nhm-v1.0-daymet-byHW-musk-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.0 model output variables using byHW-with-muskingum-mann-routing calibrated parameters with Daymet version 3 forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml:14]:
```suggestion
    description: "National Hydrologic Model version 1.0 model output variables using byHW-with-muskingum-mann-routing calibrated parameters with Daymet version 3 forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml [hunk 8]
```diff
@@ -18,7 +18,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.0/dm_byHRU_musk/nhm_v1.0_dm_byHRU_musk_combined.json'
 
-  nhm-v1.0-daymet-byHW-musk-obs-onprem:
+  nhm-v1.0-daymet-byHW-musk-obs-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.0 model output variables using byHW-with-muskingum-mann-routing-using-observed-streamflow calibrated parameters with Daymet version 3 forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml:23]:
```suggestion
    description: "National Hydrologic Model version 1.0 model output variables using byHW-with-muskingum-mann-routing-using-observed-streamflow calibrated parameters with Daymet version 3 forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml [hunk 9]
```diff
@@ -27,7 +27,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.0/dm_byHRU_musk_obs/nhm_v1.0_dm_byHRU_musk_obs_combined.json'
 
-  nhm-v1.0-daymet-byHW-noroute-onprem:
+  nhm-v1.0-daymet-byHW-noroute-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.0 model output variables using byHW-with-no-routing calibrated parameters with Daymet version 3 forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml:32]:
```suggestion
    description: "National Hydrologic Model version 1.0 model output variables using byHW-with-no-routing calibrated parameters with Daymet version 3 forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml [hunk 10]
```diff
@@ -36,7 +36,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.0/dm_byHRU_noroute/nhm_v1.0_dm_byHRU_noroute_combined.json'
 
-  nhm-v1.0-daymet-byHW-noroute_obs-onprem:
+  nhm-v1.0-daymet-byHW-noroute_obs-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.0 model output variables using byHW-with-no-routing-using-observed-streamflow calibrated parameters with Daymet version 3 forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.0-daymet-catalog.yml:41]:
```suggestion
    description: "National Hydrologic Model version 1.0 model output variables using byHW-with-no-routing-using-observed-streamflow calibrated parameters with Daymet version 3 forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml [hunk 11]
```diff
@@ -1,6 +1,6 @@
 sources:
 
-  nhm-v1.1-c404-bc-byHRU-onprem:
+  nhm-v1.1-c404-bc-byHRU-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using byHRU calibrated parameters with CONUS404 bias-corrected forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml:5]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using byHRU calibrated parameters with CONUS404 bias-corrected forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml [hunk 12]
```diff
@@ -9,7 +9,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.1/c404-bc_byHRU/nhm_v1.1_c404-bc_byHRU_combined.json'
 
-  nhm-v1.1-c404-bc-byHW-onprem:
+  nhm-v1.1-c404-bc-byHW-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using by-headwater calibrated parameters with CONUS404 bias-corrected forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml:14]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using by-headwater calibrated parameters with CONUS404 bias-corrected forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml [hunk 13]
```diff
@@ -18,7 +18,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.1/c404-bc_byHW/nhm_v1.1_c404-bc_byHW_combined.json'
 
-  nhm-v1.1-c404-bc-byHWobs-onprem:
+  nhm-v1.1-c404-bc-byHWobs-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using by-headwater-with-observed-streamflow calibrated parameters with CONUS404 bias-corrected forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-c404-bc-catalog.yml:23]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using by-headwater-with-observed-streamflow calibrated parameters with CONUS404 bias-corrected forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml [hunk 14]
```diff
@@ -1,6 +1,6 @@
 sources:
 
-  nhm-v1.1-gridmet-byHRU-onprem:
+  nhm-v1.1-gridmet-byHRU-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using byHRU calibrated parameters with gridmet forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml:5]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using byHRU calibrated parameters with gridmet forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml [hunk 15]
```diff
@@ -9,7 +9,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.1/gm_byHRU/nhm_v1.1_gm_byHRU_combined.json'
 
-  nhm-v1.1-gridmet-byHW-onprem:
+  nhm-v1.1-gridmet-byHW-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using by-headwater calibrated parameters with gridmet forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml:14]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using by-headwater calibrated parameters with gridmet forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml [hunk 16]
```diff
@@ -18,7 +18,7 @@ sources:
       storage_options:
         fo: '/caldera/hovenweep/projects/usgs/water/impd/hytest/nhm/nhm_v1.1/gm_byHW/nhm_v1.1_gm_byHW_combined.json'
 
-  nhm-v1.1-gridmet-byHWobs-onprem:
+  nhm-v1.1-gridmet-byHWobs-onprem-hw:
     driver: intake_xarray.xzarr.ZarrSource
     description: "National Hydrologic Model version 1.1 model output variables using by-headwater-with-observed-streamflow calibrated parameters with gridmet forcings"
```
Comment (amsnyder) [inline dataset_catalog/subcatalogs/nhm-v1.1-gridmet-catalog.yml:23]:
```suggestion
    description: "National Hydrologic Model version 1.1 model output variables using by-headwater-with-observed-streamflow calibrated parameters with gridmet forcings. This dataset is stored on USGS on-premise Caldera storage for Hovenweep and is only accessible via the USGS Hovenweep supercomputer."
```

File: dataset_catalog/README.md [hunk 17]
```diff
@@ -3,10 +3,10 @@ This section describes how to use HyTEST's [intake catalog](https://intake.readt
 
 Our catalog facilitates this access for HyTEST's key data offerings and is used to read the data into the notebooks contained in this repository. While intake catalogs are Python-centric, they stored as a yaml file, which should also be easy to parse using other programming languages, even if there is no equivalent package in that programming language.
```
Comment (amsnyder) [inline dataset_catalog/README.md:4]:
```suggestion
Our catalog facilitates this access for HyTEST's key data offerings and is used to read the data into the notebooks contained in this repository. While intake catalogs are Python-centric, they are stored as a yaml file, which should also be easy to parse using other programming languages, even if there is no equivalent package in that programming language.
```

**Comment by amsnyder** [comment]
Thanks @putmanannie! I made your suggested changes and will be merging this in now (meaning the intake catalog entry name for the hovenweep datasets will change and you will need to update your workflows to use the new name). I will push these changes to the JupyterBook too so they should show up there in about 15 minutes.

**Comment by putmanannie** [comment]
Thanks!

**Pull request merged**